_data/people.yml

```diff
@@ -165,6 +165,19 @@
         icon: github-alt
         name: Github
 
+- name: Lu Kissinger
+  pic: /img/people/kissingerl.jpg
+  dates: 2017-present
+  role: Graduate Research Assistant
+  description: "Lu (they/them pronouns) is a Masters student using Monte Carlo codes to investigate the feasibility of Lunar Surface Fission Power."
```

Comment:
Because you also worked as an undergrad in the group, do you want to add a sentence mentioning your previous work? e.g. "Previously Lu worked in ARFC on ___ and did a post-baccelaureate at Argonne"